x/bridge/keeper/keeper.go

```diff
@@ -797,26 +793,17 @@ func (k Keeper) CreateSnapshot(ctx sdk.Context, queryId []byte, timestamp time.T
 	}
 
 	// set snapshot by report
-	key := hex.EncodeToString(crypto.Keccak256([]byte(hex.EncodeToString(queryId) + fmt.Sprint(timestamp.Unix()))))
-	// check if map for this key exists, otherwise create a new map
-	exists, err := k.AttestSnapshotsByReportMap.Has(ctx, key)
-	if err != nil {
-		k.Logger(ctx).Info("Error checking if attestation snapshots by report map exists", "error", err)
-		return err
-	}
-	if !exists {
-		attestationSnapshots := types.NewAttestationSnapshots()
-		err = k.AttestSnapshotsByReportMap.Set(ctx, key, *attestationSnapshots)
-		if err != nil {
-			k.Logger(ctx).Info("Error setting attestation snapshots by report", "error", err)
-			return err
-		}
-	}
+	key := crypto.Keccak256([]byte(hex.EncodeToString(queryId) + fmt.Sprint(timestamp.Unix())))
 	attestationSnapshots, err := k.AttestSnapshotsByReportMap.Get(ctx, key)
-	if err != nil {
+	if err != nil && err != collections.ErrNotFound {
```

Comment:
for some reason I'm still getting this error

1:46PM INF Error getting attestation requests by height error="collections: not found: key '24' of type github.com/cosmos/gogoproto/" module=x/bridge